js/components/input-datetime.tsx

```diff
@@ -165,11 +168,10 @@ class InputDateTime extends React.Component<IInputProps, null> {
             .on('dp.change', !this.props.isDesignTime && this.onChange);
 
         this.setPickerDate(this.props.value);
-    }
 
-    componentWillUnmount() {
-        if (this.refs['datepicker'])
-            $(findDOMNode(this.refs['datepicker'])).data('DateTimePicker').destroy();
+        if (manywho.settings.global('autofocusinput', this.props.flowKey, null) && this.props.autofocusCandidate === true) {
```

Comment:
Is this if statement necessary? I didn't think it was used in the other input component implementations.

Reply:
In this element the reference is created always because is a datetime library for jquery and use the reference to communicate with it, it should be instead a react library. 
In the others elements I was checking if the reference is not null but here it will always be not null.